pybop/models/lithium_ion/weppner_huggins.py

```diff
@@ -0,0 +1,84 @@
+#
+# Weppner Huggins Model
+#
+import numpy as np
+import pybamm
+
+
+class BaseWeppnerHuggins(pybamm.lithium_ion.BaseModel):
+    """WeppnerHuggins Model for GITT.
+
+    Parameters
```

Comment:
It would be great to add a citation here. Perhaps the original 1977 paper, or a more recent one if the derivation orginated elsewhere?

Reply:
Done